glusterd2/peer/self.go

```diff
@@ -46,7 +46,6 @@ func AddSelfDetails() error {
 		Name:          gdctx.HostName,
 		PeerAddresses: []string{config.GetString("peeraddress")},
 	}
-
```

Comment:
This is unintended change and shows up in `git log` and later in `git blame`. Avoid these.